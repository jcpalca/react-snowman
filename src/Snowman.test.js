import Snowman from "./Snowman.js";
import { render, fireEvent } from "@testing-library/react";

describe("end game", function () {
  it("matches snapshot after lose", function () {
    const { container, debug } = render(<Snowman maxWrong={1} />);
    const letterButton = container.querySelector('button[value="q"]');
    // const letterButton = container.querySelector('.q');
    fireEvent.click(letterButton);
    debug(container);

    expect(container).toMatchSnapshot();
    expect(container.querySelector(".lose")).toBeInTheDocument();
  });
});
