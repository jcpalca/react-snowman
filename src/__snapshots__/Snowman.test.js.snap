// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`end game matches snapshot after lose 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="1"
      src="1.png"
    />
    <p>
      Number wrong: 
      1
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p
      class="lose"
    >
      You lose 
    </p>
  </div>
</div>
`;
